gm4_audere_shamir/data/gm4_audere_shamir/functions/main.mcfunction

```diff
@@ -0,0 +1,6 @@
+scoreboard players reset tool_max_damage gm4_ml_data
+scoreboard players reset tool_current_damage gm4_ml_data
```

Comment:
In Gamemode 4 we usually reset scores at the end of a clock cycle, this makes these values invisible to players when the scoreboard is shown.
Prefixing fake players with `$` is also recommended.